apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-menu/index.tsx

```diff
@@ -8,23 +8,24 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-
 import { LAUNCH_STORE } from '../stores';
-import { LaunchStep, LaunchSequence } from '../../../common/data-stores/launch/data';
 import LaunchMenuItem from './item';
-import './styles.scss';
 
-const LaunchStepMenuItemTitles = {
-	[ LaunchStep.Name ]: __( 'Name your site', 'full-site-editing' ),
-	[ LaunchStep.Domain ]: __( 'Select a domain', 'full-site-editing' ),
-	[ LaunchStep.Plan ]: __( 'Select a plan', 'full-site-editing' ),
-	[ LaunchStep.Final ]: __( 'Launch your site', 'full-site-editing' ),
-};
+import './styles.scss';
 
 const LaunchMenu = () => {
 	const { step: currentStep, completedSteps } = useSelect( ( select ) =>
 		select( LAUNCH_STORE ).getState()
 	);
+	const LaunchStep = useSelect( ( select ) => select( LAUNCH_STORE ).getLaunchStep() );
+	const LaunchSequence = useSelect( ( select ) => select( LAUNCH_STORE ).getLaunchSequence() );
+
+	const LaunchStepMenuItemTitles = {
+		[ LaunchStep.Name ]: __( 'Name your site', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **11** times:
`translate( 'Name your site' )` *ES Score: 9*